hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/Router.java

```diff
@@ -195,4 +197,12 @@ public static void main(String[] argv) {
       System.exit(-1);
     }
   }
+
+  public RouterClientRMService getClientRMProxyService() {
```

Comment:
VisibleForTesting?

Reply:
Thank you very much for helping to review the code, I will modify the code.